src/sentry/static/sentry/app/views/sharedGroupDetails/index.tsx

```diff
@@ -131,6 +131,7 @@ class SharedGroupDetails extends React.Component<Props, State> {
                     group={group}
                     event={latestEvent}
                     project={project}
+                    api={api}
```

Comment:
Will API requests for proguard files work on share views?

Reply:
yes I just tested it locally and it worked.

Reply:
it worked because I was logged in.. I will add a condition. thanks for pointing this out ;-)